x-pack/plugins/security_solution/public/flyout/right/components/header_title.test.tsx

```diff
@@ -31,13 +32,17 @@ moment.tz.setDefault('UTC');
 
 const dateFormat = 'MMM D, YYYY @ HH:mm:ss.SSS';
 
+const flyoutContextValue = {} as unknown as ExpandableFlyoutContext;
```

Comment:
I actually kinda like it like this better, because then it's obvious that the test actually doesn't really need anything. By using the mock, it implies that the test actually needs something defined, which in this case it doesn't.

I don't have a strong opinion though, so if for consistency you'd prefer using the mock everywhere I'll change it!

Reply:
i agree that it's important to show how context impact what we expect in test. I think the mock context is basic enough and at the same time type safe, so we can minimize assertion to unknown

Reply:
this will be handled as part of a separate PR that will cleanup all the mocks for Jest and Storybook